components/Carousel/Carousel.js

```diff
@@ -1,4 +1,53 @@
+class CarouselItem {
+	constructor(element) {
+		this.element = element;
+	}
+
+	select() {
+		this.element.classList.add("Carousel__item-focused");
+	}
+
+	deselect() {
+		this.element.classList.remove("Carousel__item-focused");
+	}
+}
+
+
 class Carousel {
```

Comment:
Wow, your carousel implementation is so beautiful! Love how you made it flow smoothly.